jobs-builder/jobs/dependencies.yaml

```diff
@@ -0,0 +1,66 @@
+# This file contain the configurations to generate Kernel jobs.
+---
+- job-template:
+    name: 2.x-kernel-{flavor}-{arch}
+
+    # Template default variables
+    #
+    arch: x86_64
+    os: ubuntu1804
```

Comment:
Why not 20.04?

Reply:
The QEMU and Kernel jobs are running on Ubuntu 18.04. I don't want to change it now.